Makefile

```diff
@@ -283,6 +285,18 @@ api-ref-docs:
 		--renderer=markdown \
 		--output-path=${PWD}/site-src/reference/spec.md
 
+# Generate a virtualenv install, which is useful for hacking on the
```

Comment:
Added target to make a virtualenv, copied from the Gateway API project

Reply:
yeah, it wouldn't be needed for the local preview. I left it in for if people are doing local development on the docs theme. For example, if you needed to add some other `mkdocs` plugins down the road, or if you wanted to manually set the versions with `mike` (such as to remove an old version), it could be useful.

wdyt?